tests/conftest.py

```diff
@@ -310,11 +309,33 @@ def metadata_path():
 @pytest.fixture(scope="session")
 def entity_types():
     """Entity types that can be used throughout tests."""
-    request_path = (
-        ROOT_PATH / "tests" / "data" / "mining" / "request" / "ee_models_library.csv"
-    )
+    return ["CHEMICAL", "ORGANISM"]
+
 
-    return list(pd.read_csv(request_path)["entity_type"].unique())
+@pytest.fixture()
+def spacy_model_path(tmpdir, entity_types):
+    """Structure of files for entity models."""
+    data_dir = Path(tmpdir) / "some" / "path"
+    models_dir = data_dir / "models" / "ner_er"
+    models_dir.mkdir(parents=True)
+
+    for etype in entity_types:
+        etype_model_dir = models_dir / f"model-{etype.lower()}"
+        etype_model_dir.mkdir()
+        meta_path = etype_model_dir / "meta.json"
+        with open(meta_path, "w") as f:
+            f.write(
+                json.dumps(
```

Comment:
I think this can be simplified to `json.dump(data, fp)`

Reply:
Done in 7f5c0d6.